doc/GuideForCommitters.md

```diff
@@ -93,19 +93,19 @@ build, add a comment to the pull request that follows the syntax:
 Where `{platform-list}` is a comma-separated list of platforms chosen from the
 following tags:
```

Comment:
Since you renamed "tag" in the column heading below, should "tags" now be "aliases" ?

Reply:
Updated.